Content.Server/StationEvents/Events/MouseMigration.cs

```diff
@@ -0,0 +1,50 @@
+using System.Linq;
+using Content.Server.StationEvents.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.StationEvents.Events;
+
+public sealed class MouseMigration : StationEvent
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+
+    public static List<string> SpawnedPrototypeChoices = new List<string>() //we double up for that ez fake probability
+        {"MobMouse", "MobMouse1", "MobMouse2", "MobRatServant"};
+
+    public override string Name => "MouseMigration";
+
+    public override string? StartAnnouncement =>
+        Loc.GetString("station-event-mouse-migration-announcement");
+
+    public override int EarliestStart => 30;
+
+    public override int MinimumPlayers => 35; //this just ensures that it doesn't spawn on lowpop maps. 
+
+    public override float Weight => WeightVeryLow;
```

Comment:
Can probably be a mere WeightLow, as it starts at minimum 30 min in and as such already has gimped odds.